nandtool/nandtool.c

```diff
@@ -177,6 +177,91 @@ static int nandtool_check(void)
 }
 
 
+static int nandtool_dump(const char *outpath, unsigned int start, unsigned int nblocks, int oob)
+{
+	flashsrv_info_t *info;
+	int ret;
+	size_t blocksz, partsz, pagesz;
+	size_t addr, endaddr;
+	uint8_t *buf;
+	int outfd;
+
+	outfd = open(outpath, O_RDWR | O_CREAT);
```

Comment:
`O_WRONLY` ?

Reply:
Yes, `O_WRONLY | O_CREAT` is sufficient.